console.assert(1==2, 'raise an exception');
